pages/get_involved.js

```diff
@@ -65,12 +66,15 @@ export default () => (
           development, Operation Code is always looking for supporters like you.
         </p>
 
-        <LinkButton href="/who_we_serve">Learn More</LinkButton>
+        <LinkButton className="hero_banner_link_button" href="/who_we_serve">
```

Comment:
You've added this class, but we use CSS Modules, meaning you should be stemming from an existing `styles` import that has an existing class on it's stylesheet. https://github.com/css-modules/css-modules

In other words, `className="hero_banner_link_button"` does nothing.